CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ message(STATUS "Configuration: ${CONFIG_OPTIONS}")
 
 project(openscad)
 
+set(CMAKE_CXX_FLAGS "-O2")
```

Comment:
This should be turned off when in debug mode. I thought cmake managed that automatically..

Reply:
If you set `CMAKE_CXX_FLAGS_RELEASE` instead, cmake will automatically merge your params into `CMAKE_CXX_FLAGS` based on what compilation mode is selected through `CMAKE_BUILD_TYPE`. See https://cmake.org/Wiki/CMake_Useful_Variables#Compilers_and_Tools.

Also note that setting a variable directly clobbers its previous value. What you probably want is to *append* "-O2", so you want something like `set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O2")`